gamelogs-client/src/App.tsx

```diff
@@ -1,35 +1,171 @@
+  return (  
+    <div className="w-screen h-screen">
+      <main className="h-full">
+        
+        {/*Hero Section*/}
+        <section className="relative h-4/5">
+          <div className="absolute inset-0 bg-black/50 z-10"></div>
+          <div className="absolute inset-0 flex flex-col justify-center items-center z-10">
+            <h1 className="text-9xl font-black text-white">GameLogs</h1>
+          </div>
+            <div className="absolute top-0 w-full h-full flex flex-col gap-6 z-0 justify-center">
+              <div className="flex gap-4 animate-scroll-right">
+                {[...games, ...games].map((game, idx) => (
+                  <div key={idx} className="h-36 py-2 px-4 rounded-lg" style={{ minWidth: '14rem', backgroundColor: game.thumbnail }}>
+                    <div className="flex flex-col justify-end h-full">
+                      <h3 className="text-md font-regular flex justify-between">
+                        <span>{game.title}</span>
+                        <span>{game.logo}</span>
+                      </h3>
+                    </div>
+                  </div>
+                ))}
+              </div>
+              <div className="flex gap-4 animate-scroll-left">
+                {[...games, ...games].map((game, idx) => (
+                  <div key={idx} className="h-36 py-2 px-4 rounded-lg" style={{ minWidth: '14rem', backgroundColor: game.thumbnail }}>
+                    <div className="flex flex-col justify-end h-full">
+                      <h3 className="text-md font-regular flex justify-between">
+                        <span>{game.title}</span>
+                        <span>{game.logo}</span>
+                      </h3>
+                    </div>
+                  </div>
+                ))}
+              </div>
+              <div className="flex gap-4 animate-scroll-right">
+                {[...games, ...games].map((game, idx) => (
+                  <div key={idx} className="h-36 py-2 px-4 rounded-lg" style={{ minWidth: '14rem', backgroundColor: game.thumbnail }}>
+                    <div className="flex flex-col justify-end h-full">
+                      <h3 className="text-md font-regular flex justify-between">
+                        <span>{game.title}</span>
+                        <span>{game.logo}</span>
+                      </h3>
+                    </div>
+                  </div>
+                ))}
+              </div>
+              <div className="flex gap-4 animate-scroll-left">
+                {[...games, ...games].map((game, idx) => (
+                  <div key={idx} className="h-36 py-2 px-4 rounded-lg" style={{ minWidth: '14rem', backgroundColor: game.thumbnail }}>
+                    <div className="flex flex-col justify-end h-full">
+                      <h3 className="text-md font-regular flex justify-between">
+                        <span>{game.title}</span>
+                        <span>{game.logo}</span>
```

Comment:
These can likely be their separate components. Would help since it's repeated a decent bit. Also helps with readability.